tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/AllTests.java

```diff
@@ -24,7 +24,8 @@
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
 	AllNonBrowserTests.class,
-	AllBrowserTests.class
+	AllBrowserTests.class,
```

Comment:
The rest of the o.e.swt.custom tests are in AllWidgetTests. Would you please put that one there too?

Reply:
`BusyIndicator` is not a widget :-)

Reply:
Neither is StyleRange or Monitor , this is just history. I don't mind these being rearranged but until such effort is started let's keep things aligned.

Reply:
Done!